src/SelectCatalog.js

```diff
@@ -181,3 +251,4 @@ const Label = ({text, selectId, isEnhanced}) => isEnhanced ? (
 );
 
 export default SelectCatalog;
+export {Select};
```

Comment:
Put `export` before class above instead? I think that's what we did in the other instances where we're reusing these?